### PR TITLE
Enable Nova Protocol on Ledger Nano App

### PR DESCRIPTION
**Pull request by shufps**
Enabled Nova Protocol for Ledger Nano

### CODE REVIEW

File: sdk/src/client/secret/ledger_nano.rs [hunk 1]
```diff
@@ -159,7 +160,7 @@ impl SecretManage for LedgerSecretManager {
         let lock = self.mutex.lock().await;
 
         // get ledger
-        let ledger = get_ledger(coin_type, bip32_account, self.is_simulator).map_err(Error::from)?;
+        let ledger = get_ledger(coin_type, Protocols::Nova, bip32_account, self.is_simulator).map_err(Error::from)?;
```
Comment (thibault-martinez) [inline sdk/src/client/secret/ledger_nano.rs]: Nothing that should be blocking now but if ever you want to change in a future version: enum names are usually singular in Rust

Comment (thibault-martinez) [reply sdk/src/client/secret/ledger_nano.rs]: And I would personally put that parameter first

**Review by thibault-martinez — DISMISSED** [review]

**Review by Thoralf-M — DISMISSED** [review]

File: sdk/src/client/secret/ledger_nano.rs (hunk 1 above)
Comment (shufps) [reply sdk/src/client/secret/ledger_nano.rs]: Ok, changed :blush:

**Comment by shufps** [comment]
ah something happend ... sorry have to revert my last commit, it cargo fmted everything :scream:

**Review by Thoralf-M — APPROVED** [review]

**Review by thibault-martinez — APPROVED** [review]

File: sdk/src/client/secret/ledger_nano.rs (hunk 1 above)
Comment (thibault-martinez) [reply sdk/src/client/secret/ledger_nano.rs]: 🙏🏻

**Pull request merged**